modules/tiled-gallery/tiled-gallery/tiled-gallery.js

```diff
@@ -42,28 +44,37 @@
 
 	TiledGallery.prototype.addCaptionEvents = function () {
 		// Hide captions
-		this.gallery.find( '.tiled-gallery-caption' ).hide();
+		var galleryCaptions = this.gallery.querySelectorAll( '.tiled-gallery-caption' );
+		Array.prototype.forEach.call( galleryCaptions, function ( el ) {
+			el.style.display = 'none';
+		} );
 
-		// Add hover effects to bring the caption up and down for each item
-		this.gallery.find( '.tiled-gallery-item' ).hover(
-			function () {
-				$( this ).find( '.tiled-gallery-caption' ).stop( true, true ).slideDown( 'fast' );
-			},
-			function () {
-				$( this ).find( '.tiled-gallery-caption' ).stop( true, true ).slideUp( 'fast' );
+		var mouseHoverHandler = function ( e ) {
+			var itemEl = e.target.closest( '.tiled-gallery-item' );
+			var displayValue = 'mouseover' === e.type ? 'block' : 'none';
+
+			if ( itemEl ) {
+				var itemCaption = itemEl.querySelector( '.tiled-gallery-caption' );
+				if ( itemCaption ) {
+					itemCaption.style.display = displayValue;
```

Comment:
I have removed the animation (`slideDown` and `slideUp`). Instead, the caption just appears and hides with no animation. We can add the animation back in if necessary, but this minor degradation felt efficient and non-consequential.